src/psij/executors/batch/batch_scheduler_executor.py

```diff
@@ -373,7 +375,9 @@ def parse_status_output(self, out: str) -> Dict[str, JobStatus]:
         this method need not process the *exit code file* or the *script output file* since it is
         done by the base `BatchSchedulerExecutor` implementation.
         """
-        pass
+        if exit_code != 0:
+            raise RuntimeError(f"status command exited with {exit_code} and output {out!r}")
```

Comment:
Do we want to be consistent about using double/single quotes for strings?

Reply:
> It nags me that this may belong to a separate method. And when I say "it nags me", it's not quite clear to me that it should.

Do you mean I should break this method out into a free function or something? That would probably be better, I will fix that.

> Do we want to be consistent about using double/single quotes for strings?

Yeah I think so, although I would like to have a style check tell us when we make a mistake and a formatter to fix it. Until then I would let it be chaos.

Reply:
> Yeah I think so, although I would like to have a style check tell us when we make a mistake 

https://github.com/zheller/flake8-quotes

We can add that.

> and a formatter to fix it. 

That, I think, is best left to each's coding environment, although we could provide a tool that automates some of it. That said, we did have a bit of this discussion in the past and I'd personally like to avoid going through the same points again unless there's enough of a change in the background story to warrant it. But that's me.

> Until then I would let it be chaos.

I'm not entirely sure that messing things up on purpose to force a certain outcome is an effective way to solve this problem :)

Maybe we can talk about it during the meeting.

Reply:
> Do you mean I should break this method out into a free function or something? That would probably be better, I will fix that.

Protected method in BatchSchedulerExecutor I was thinking, but yeah.

Reply:
Fixed, squashed, and pushed. I also changed to single quotes because you're right the rest of the file is single quotes. Although I didn't change the double quotes in files with double quotes.

> Until then I would let it be chaos.

I didn't mean intentionally messing things up, I think as far as quotes go, letting it be chaos (i.e. just do whatever) is a fine long-term approach. About some other style things (like line length) I wouldn't say the same.

Reply:
> About some other style things (like line length) I wouldn't say the same.

We have it set to 100 in .flake8. We can discuss, but I think it's been a while since the 80 column terminals were the norm.